src/ic-cdk/src/api/management_canister/main/types.rs

```diff
@@ -198,9 +198,9 @@ pub struct InstallChunkedCodeArgument {
     /// Principal of the canister being installed
     pub target_canister: CanisterId,
     /// The canister in whose chunk storage the chunks are stored (defaults to target_canister if not specified)
-    pub storage_canister: Option<CanisterId>,
+    pub store_canister: Option<CanisterId>,
     /// The list of chunks that make up the canister wasm
-    pub chunk_hashes_list: Vec<ChunkHash>,
+    pub chunk_hashes_list: Vec<Vec<u8>>,
```

Comment:
This will be serialized inefficiently when using serde (it'll be fine when using candid, I think?).
We should either use `ByteBuf` or manually implement serialize/deserialize for this field.

Reply:
Yeah ideally we'd expose `Vec<u8>` rather than `ByteBuf`, but when its `Vec<Vec<u8>>` we can't use the `#[serde(with = "serde_bytes")]` trick. I think the only way to avoid exposing `ByteBuf` would be to manually implement serialize and deserialize.